src/utils/setAuthToken.ts

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+const setAuthToken = (token: any) => {
+     if(token) {
+          axios.defaults.headers.common['x-auth-token'] = token;
+     } else {
+          delete axios.defaults.headers.common['x-auth-token'];
+     }
+}
```

Comment:
Implicit function behavior. Function name said me that it should set token. But if I didn't provide. any param it deletes token from headers...Split this function to two functions

Reply:
fixed